Dapper/SqlMapper.DapperRow.cs

```diff
@@ -46,14 +46,14 @@ internal bool TryGetValue(int index, out object value)
             {
                 if (index < 0)
                 { // doesn't exist
-                    value = null;
+                    value = null!;

```

Comment:
I think we should represent this as nullable throughout the class, e.g. on TryGetValue, e.g. `[NotNullWhen(true)]` on `TryGetValue` and `object?` throughout - thoughts?